README.md

```diff
@@ -1,86 +1,120 @@
+General:
+* Three group of users will be directed to different homepages when they log into the app. 
+* Lower power group of users don't have access to the pages that higher power group of users use. 
+
+##Part2 File Tree(partial)
 ```
-##Part1 Setup
+.
+├── LICENSE             - license for this app
+├── Notes.md            - development notes
+├── README.md           - this file
+├── __main__.py         - the script used to run the app
+├── config.cfg          - the app's configuration file
+├── db_populate.py      - add initial values in to the database(mainly for testing)
+├── db_reset.py         - reset and repopulate the database
+├── error_log           - contains error log file
+├── lemur
+│   ├── __init__.py     - initialize the app and set up global variables
+│   ├── static
+│   │   ├── css         - contain css files for templates' decoration
+│   │   └── js          - contain javascripts for templates, library, utility functions or global constants
+│   ├── templates       - contains html templates
+│   ├── utility.py      - utility functions used to make view functions in views.py more readable
+│   └── views.py        - view functions for all the templates
+├── migrations          - contains files for database migration
+├── requirements.txt    - required modules/libraries for the app
+├── run.py              - the script used to migrate/upgrade the database
+└── unit_test.py        - unit test for the app
+```
+##Part3 Setup
 ###Step1 Create Virtual Environment for All Required Dependencies
 ``` mkvirtualenv lemur --python=python3```
 
-###Step2 Install Required Packages: Flask, psycopg2, SQLAlchemy, WTForms, Flask-WTF
+###Step2 Install Required Packages
 ```
-pip install -r requirement
+pip install -r requirements.txt
 ```
 
 ###Step3: Create Local Database
-We’ll need to get a Postgres database set up to store our todo list. We’ll also add Python ORM SQLAlchemy to our app. Once you have Postgres installed, create a database and name it lemur to use as a local database.<br\>
-* Run PostgreSQL command line client.<br\>
-* Create a database user with a password.<br\>
-* Create a database instance.<br\>
+We’ll need to get a Postgres database set up to store our todo list. We’ll also add Python ORM SQLAlchemy to our app. Once you have Postgres installed, create a database and name it lemur to use as a local database. <br>
+* Run PostgreSQL command line client.
+* Create a database user with a password.
+* Create two database instances(one for the app and the other for testing).
 
 ```
 psql
 create user zzy with password 'mypassword';
 create database lemur owner zzy encoding 'utf-8'; 
+create database lemur_test owner zzy encoding 'utf-8'; 
 ```
 
-###Step4: Download and Link boostrap library, jquery-ui 1.11.4 library, and jquery-2.1.3 library
+###Step4: Setup Database
+In /lemur
+```
+python3 db_create
+```
 
-###Step5: Run the app `python3 myapp`
+###Step5: Ask zhongzi@reed.edu for the private configuaration file
```

Comment:
what should people do once you are no longer a student? I'd recommend working with IT to set up an email address that gets forwarded to the group of people with the configs.